### DISCUSSION TITLE
Github wiki: problems with authentication when pushing changes from local to remove

### DISCUSSION THREAD

**Discussion by fguillen** [community/community#130434] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have cloned the wiki of my repo locally. I prefer to edit it with my favorite editor.

I make the changes and commit them, but when I try to push them to remote, I have trouble with authentication. 

The first surprise is that the `git push` command asked me for credentials. Since all my users have SSH keys, I don't need to write credentials for every push.

Ok, I put my credentials, but then I see this error:

```
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
```

Reading this documentation doesn't clarify anything. In summary, I can not authenticate.

How can I push my changes on the GitHub wiki from local to remote?

**Comment by fguillen** [comment]
Ok, I have created a "ssh" URL version for the wiki repo, and there I can push:

```
origin	https://github.com/fguillen/fantasy.wiki.git (fetch)
origin	https://github.com/fguillen/fantasy.wiki.git (push)
origin_ssh	git@github.com:fguillen/fantasy.wiki.git (fetch)
origin_ssh	git@github.com:fguillen/fantasy.wiki.git (push)
```

**Comment by fguillen** [comment]
This should be set somewhere in the documentation, or in the wiki page the repo URL to close should be the `git` one not the `https` one:

![Screenshot 2024-06-29 at 23 06 36](https://github.com/community/community/assets/23043/4b563152-16c0-4f7b-9a97-ea49653bdc93)

**Comment by Rishi-Sudhakar** [comment]
Well, you have two origins right now. You have to change your remote first, and set it as your remote. (you don't need to create an additional one)
```
git remote set-url origin git@github.com:fguillen/fantasy.wiki.git
```

now check 

```
git remote -v
```

now try pushing again

**Reply by fguillen** [reply]
Yes, I understand this. Showing my 2 origins was just to show the difference. I'm sorry it was confusing.

What I mean is that in the web interface, when I copy the repo URL, it gives the `https` version and not the `ssh` version. Since the `https` version is no longer valid, the web interface should give me the `ssh` version directly.
